pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -393,6 +410,9 @@ class ToolReturnPart:
     tool_call_id: str
     """The tool call identifier, this is used by some models including OpenAI."""
 
+    extra_data: Any = None
+    """Additional programmatic data that can be accessed by the application but is not sent to the LLM."""
```

Comment:
"programmatic data" is not really clear to me, how about just "Additional data that can be accessed programmatically..."?